apps/els_dap/src/els_dap.erl

```diff
@@ -128,6 +137,7 @@ configure_logging() ->
                                              , "[", level, "] "
                                              , msg, "\n"
                                              ]
+                              %  , single_line => false
```

Comment:
Leftover?

Reply:
Yes, single-line comments are too hard to parse for me when debugging :)